test/param/number-list.spec.js

```diff
@@ -41,4 +41,54 @@ describe('Testing NumberList Parameter', () => {
       }
     })).to.throw('Invalid Value for json-Parameter "list" provided.');
   });
+
+  describe('Testing optional parameter "minItemValue"', () => {
+    let queryParamInValidMin;
+    let bodyParamInValidMin;
+    before(() => {
+      queryParamInValidMin = api.NumberList('list', 'query', { minItemValue: 1 });
+      bodyParamInValidMin = api.NumberList('list', 'json', { minItemValue: 1 });
+    });
+
+    it('Testing invalid min-value of list item in query param', () => {
+      expect(() => queryParamInValidMin.get({
+        queryStringParameters: {
+          list: '[0, 1, 200]'
+        }
+      })).to.throw('Invalid Value for query-Parameter "list" provided.');
+    });
+
+    it('Testing invalid min-value of list item in json param', () => {
+      expect(() => bodyParamInValidMin.get({
+        body: {
+          list: [0, 1, 200]
+        }
+      })).to.throw('Invalid Value for json-Parameter "list" provided.');
+    });
+  });
+
+  describe('Testing optional parameter "maxItemValue"', () => {
```

Comment:
I would have a third test showing that if all items in the list are smaller than `maxItemValue`, you will not get an error.